english/myExGf/myExGf.md

```diff
@@ -0,0 +1,9 @@
+She's sweet and pretty and smart, but she used to chew like Jabba-the-hut. I couldn't tell her this because I loved her. I really wanted to end the relationship, but I couldn't tell her why. Then one day, we were watching a Star Wars marathon when Jabba-the-hut came on and started flapping his gums and smacking his lips and devouring everything. It was disgusting... 

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.